internal/tcp/tcp.go

```diff
@@ -522,7 +524,8 @@ func Receiver(
 				// Received SYN+ACK (Open port)
 				logger.Debug("Received",
 					zap.String("flag", "SYN ACK"),
-					zap.Object("port", pkt.srcPort))
+					zap.String("src_address", fromAddrStr),
+					zap.Any("src_port", pkt.srcPort))
```

Comment:
The docs actually over state things:
- `zap.Any` is just a big type assertion https://github.com/uber-go/zap/blob/master/field.go#L233
- the final case there uses reflection, but everything else has a much faster path
- now, this is micro slow because it needs to allocate an `uint16` on the heap to pass it as interface{} (until go 1.9)
- so yeah, probably stop using `zap.Any` in hot paths for sure, but the reason isn't primarily "reflection" in this case